crates/lib/src/install/config.rs

```diff
@@ -191,11 +192,21 @@ impl InstallConfiguration {
 #[context("Loading configuration")]
 /// Load the install configuration, merging all found configuration files.
 pub(crate) fn load_config() -> Result<Option<InstallConfiguration>> {
+    load_config_at("/")
+}
+
+pub(crate) fn load_config_at<P: AsRef<Path>>(root_dir: P) -> Result<Option<InstallConfiguration>> {
```

Comment:
Totally fine as is but I think it's slightly more idiomatic now to do `load_config_at(root_dir: impl AsRef<Path>)` - it's less typing

Reply:
Aha, indeed, that is nicer (still not using rust much which is a bit of a shame)! Thanks!